src/components/shared/MarkerTooltipContents.js

```diff
@@ -553,6 +553,24 @@ function getMarkerDetails(
               timings.phase_times,
               6
             );
+            let gcsize;
+            const post_heap_size = timings.post_heap_size;
+            if (post_heap_size !== undefined) {
+              gcsize = _markerDetail(
+                'gcsize',
+                'Heap size',
+                formatBytes(timings.allocated_bytes) +
+                  ' - ' +
```

Comment:
What does this dash represent? My understanding from reading the code is that this shows the pre and post heap size. I think maybe this label should be adjusted to make that more obvious.

Reply:
That makes sense. I've added another patch.